layouts/shortcodes/otel-custom-instrumentation.md

```diff
@@ -0,0 +1,9 @@
+Custom instrumentation allows programmatic creation, modification, or deletion of traces to send to Datadog, over and above the default automatic instrumentation from the tracing SDKs. This section describes how to custom instrument your code to generate Datadog spans and tags using the [OpenTelemetry (OTel) API][101].
+
+This custom instrumentation provides an alternate way to custom (manually) instrument your code. The Datadog tracing libraries implement the OTel API spec with the OTel Datadog Trace Provider, which processes the telemetry with the tracing library and sends it to Datadog. You can use this approach, for example, if your code has already been instrumented with OTel and you want to gain the benefits of using the Datadog tracing libraries without changing your code.
```

Comment:
I wouldn't suggest calling out the Tracer Provider here in the docs, and leave that up to the individual languages to describe if it makes sense for them. Not every language implemented this in exactly the same way, so that's more of an implementation detail. The important piece is that Datadog provides an OpenTelemetry compatible API that can be used to instrument your code and send traces to Datadog.